pytorch_toolkit/formula_recognition/im2latex/data/utils.py

```diff
@@ -291,7 +293,7 @@ class TransformDilation:
     """Morphologic dilation
     """
 
-    def __init__(self, kernel_size=3, iterations=3):
+    def __init__(self, kernel_size=3, iterations=3, **kwargs):
```

Comment:
Vladimir, you have added `**kwargs` to all these transform classes.

Is it really required?  
I'd better make these `__init__` methods without `**kwargs`, it will check errors in the config file -- when a user points a wrong parameter to a transform it is a good idea to report it